Samples/Server/Unity-FractalStream/Assets/Scripts/StreamingUnityServerPlugin.cs

```diff
@@ -0,0 +1,159 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using UnityEngine;
+using UnityEngine.Rendering;
+
+namespace Microsoft.Toolkit.ThreeD
+{
+    /// <summary>
+    /// Represents the streaming unity server plugin
+    /// </summary>
+    public class StreamingUnityServerPlugin : IDisposable
+    {
+        /// <summary>
+        /// The name of the plugin from which we dllimport
+        /// </summary>
+        private const string PluginName = "StreamingUnityServerPlugin";
+
+        /// <summary>
+        /// A representation of the marshalled function that native code calls when input is received
+        /// </summary>
+        /// <param name="inputData">the input message data</param>
+        public delegate void InputHandlerDelegate(string inputData);
+
+        /// <summary>
+        /// A representation of the marshalled function that native code calls when a debug message is logged
+        /// </summary>
+        /// <param name="logLevel">the severity at which the message occurs</param>
+        /// <param name="logData">the data of the message</param>
+        public delegate void LogHandlerDelegate(int logLevel, string logData);
+
+        #region Dll Imports
+
+#if (UNITY_IPHONE || UNITY_WEBGL) && !UNITY_EDITOR
+	[DllImport ("__Internal")]

```

Comment:
Nitpick: wrong indentation.

Reply:
? the `DllImport` matches the indentation of the function definition, do you feel it should not?

Reply:
Yeah, it doesn't look right to me when viewing in Visual Studio or Notepad++.